src/components/questions/questionsList.js

```diff
@@ -40,19 +39,22 @@ constructor(props){
   render(){
     return (
     <React.Fragment>
-      <h1 className="title">QUESTONS & ANSWERS</h1>
+      <h1 className="title">QUESTIONS & ANSWERS</h1>
       <div className="container questionsExpand">
         {
         (this.props.store.questions[0] ?
           this.determineDisplayed(this.props.store.questions).map( (question, index) => {
```

Comment:
I did a similar conditional rendering of data in my list component for rendering my related product cards. Helped simplify the number of steps needed to get the data on the DOM.